### DISCUSSION TITLE
question(theme): 当处于暗黑模式并刷新页面后模式信息未保存

### DISCUSSION THREAD

**Discussion by j4587698** [dotnetcore/BootstrapBlazor#4292] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![Z6X( TY7{I835 ZT1MSDJLV](https://github.com/user-attachments/assets/9aa08b8c-acfb-443b-b09a-957b3248560a)
![2P@JDCEW@RA}0 J1V@ER{ R](https://github.com/user-attachments/assets/a34135d9-e123-4da4-a2f7-3637db0adbd4)
![Z~FYY(B%B `28B8@$ D28(A](https://github.com/user-attachments/assets/f01bf0a5-2417-41cc-b6e7-1a8afee624f2)

ThemeProvider组件的模式与实际模式不符。

### Steps To Reproduce

https://github.com/j4587698/TableTest
就看ThemeProvider即可，这是上次table测试的例子里加的

**Comment by bb-auto** [comment]
@j4587698 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply, marked as answer]
@j4587698 

```
<head>
    <meta charset="utf-8"/>
    <title>Bootstrap Blazor Server App</title>
    <ThemeLoader></ThemeLoader>
</head>
```

增加 `ThemeLoader`  组件即可
